components/DelegatingWidget/index.tsx

```diff
@@ -11,11 +11,16 @@ import InputBox from "./InputBox";
 import ProjectionBox from "./ProjectionBox";
 import { Tab, TabList, Tabs } from "./Tabs";
 
+// Define a type for either a Transcoder or a Delegate.
+export type TranscoderOrDelegateType =
```

Comment:
@victorges this is not the most optimal experience but it fixes the vercel build for now. I don't have time to do a full cleanup of this logic sadly.